### ISSUE TITLE
Resize input source before tracking

### ISSUE THREAD

**Issue by badkagola** [SkalskiP/yolov8-native-tracking#8]
source = "test.mp4" # my input video resolution is (2304 * 1296) 

The source is being used in the following code line:
for result in model.track(source=source, show=False, stream=True, agnostic_nms=True):#verbose=False):

I want to resize it like - frame = imutils.resize(frame, width=600) before passing input to the frame. Can you please help with this?

**Comment by SkalskiP** [comment]
Hi @badkagola 👋🏻 ! So you would like to use a different resolution of the frame from the camera?

**Comment by badkagola** [comment]
> Hi @badkagola 👋🏻 ! So you would like to use a different resolution of the frame from the camera?

Yes. How can it be done? I would really appreciate your help on this!

**Comment by SkalskiP** [comment]
It looks like you can do it this way: `model.track(source=source, show=False, stream=True, agnostic_nms=True, imgsz=600):`.
